common/beerocks/bcl/source/beerocks_socket_thread.cpp

```diff
@@ -170,6 +170,13 @@ bool socket_thread::handle_cmdu_message_uds(Socket *sd)
     if (available_bytes != message_size) {
```

Comment:
Sorry, but I don't understand how these changes match the commit description.

If this condition is true is because readBytes() could not read the whole message. According to `recv` man page and the MSG_WAITALL flag description, this can happen if a signal is caught or an error or disconnect occurs, not because message is very large.

A *very large message* is a message bigger than the buffer size so I was expecting something like `if (message_size > sizeof(rx_buffer))` somewhere in this method.